index.js

```diff
@@ -109,15 +111,18 @@ function runNgrok(opts, cb) {
 			{cwd: __dirname + '/bin'});
 
 	ngrok.stdout.on('data', function (data) {
+		if (stopListening) return;
 		var msg = data.toString();
 		var addr = msg.match(ready);
 		if (addr) {
+			stopListening = true;
```

Comment:
good catch! same should be done inside ngrok.stderr listener
did you consider using ```removeAllListeners``` insted of introducing a flag?
```
ngrok.stdout.on('data', function (data) {
		var msg = data.toString();
		var addr = msg.match(ready);
		if (addr) {
			api = request.defaults({
				baseUrl: 'http://' + addr[1],
				json: true
			});
			done();
		} else if (msg.match(inUse)) {
			done(new Error(msg.substring(0, 10000)));
		}
	});

	ngrok.stderr.on('data', function (data) {
		var info = data.toString().substring(0, 10000);
		done(new Error(info));
	});


function done(err) {
  ngrok.stdout.removeAllListeners('data');
  ngrok.stdout.removeAllListeners('data');
  cb(err);
} 
```